weaviate/schema/crud_schema.py

```diff
@@ -764,6 +769,72 @@ def _create_classes_with_primitives(self, schema_classes_list: list) -> None:
         for weaviate_class in schema_classes_list:
             self._create_class_with_primitives(weaviate_class)
 
+    def create_class_tenants(self, class_name: str, tenants: Union[dict, str]) -> None:
+        """
+        Create classes tenants in Weaviate.
```

Comment:
how does this differ from `create_class`? Does this API create tenants for an existing class?

Reply:
This method adds tenants to an existing class, if class doesn't exist then an error is being thrown